{{cookiecutter.project_folder}}/pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[tool.poetry]
+name =  "{{ cookiecutter.namespace + '-' + cookiecutter.module_name if cookiecutter.namespace else cookiecutter.module_name }}"
+version = "{{ cookiecutter.version }}"
+description = "{{ cookiecutter.short_description }}"
+authors = ["{{ cookiecutter.full_name }} <{{ cookiecutter.email }}>"]
+readme = "README.md"
+packages = [{include =  "{{ cookiecutter.namespace + '/' + cookiecutter.module_name if cookiecutter.namespace else cookiecutter.module_name }}"}]
+
+[tool.poetry.dependencies]
+python = ">={{ cookiecutter.python_version }}, <3.13"
+{%- if cookiecutter.use_jupyterlab == "y" %}
+jupyterlab = "^3.0.0"
+{%- endif %}
+{%- if "{{ cookiecutter.command_line_interface|lower }}" == "cli" %}
```

Comment:
Can we enable this always similar to pydantic and fastapi?

Reply:
Same as the comment above. CLI can be an option.